### PR TITLE
MATX-236 - Code documentation and some cleanup.

### PR DESCRIPTION
**Pull request by ppenenko**

### CODE REVIEW

File: source/MaterialXContrib/MaterialXMaya/MayaUtil.cpp [hunk 1]
```diff
@@ -44,5 +44,37 @@ void registerFragment(const std::string& fragmentName, const std::string& fragme
     }
 }
 
+void TextureDeleter::operator()(MHWRender::MTexture* texture)
```
Comment (ppenenko) [inline source/MaterialXContrib/MaterialXMaya/MayaUtil.cpp]: Moved to MayaUtil from anonymous namespace for better discoverability.

File: source/MaterialXContrib/MaterialXMaya/README.md [hunk 2]
```diff
@@ -20,16 +20,17 @@ You can now create MaterialX nodes in your scene.
...
-A `MaterialXTextureNode` can be mapped to any shader.
+A `MaterialXTextureNode` can be connected to any surface node.
 For example, you can map its OutColor attribute to the Color attribute of a Lambert, or the Specular Color of an aiStandardSurface. 
 
 Both node types expose the following attributes:
 *   `documentFilePath`: the file path to the MaterialX document (an XML file with extension `.mtlx`).
 *   `elementPath`: the path to the MaterialX element within the document, in a slash-delimited format similar to XPath, e.g. <material name/element name>. 
-*   `environmentRadianceMap`: the file path to the environment map to use for specular shading. Defaults to a texture that ships with the plug-in.
-*   `environmentIrradianceMap`: the file path to the environment map to use for diffuse shading. Defaults to a texture that ships with the plug-in.
+*   `environmentRadianceMap`: the file name of the environment map to use for specular shading. Defaults to a texture that ships with the plug-in.
```
Comment (ppenenko) [inline source/MaterialXContrib/MaterialXMaya/README.md]: Technically, these aren't paths.

File: source/MaterialXContrib/MaterialXMaya/CreateMaterialXNodeCmd.h [hunk 3]
```diff
@@ -14,21 +17,25 @@ namespace mx = MaterialX;
 namespace MaterialXMaya
 {
 
-///
-///
+/// @class CreateMaterialXNodeCmd
+/// Creates one or many MaterialX nodes from the specified MaterialX document.
```
Comment (bernardkwok) [inline source/MaterialXContrib/MaterialXMaya/CreateMaterialXNodeCmd.h]: one or *more* instead of many?

Comment (ppenenko) [reply source/MaterialXContrib/MaterialXMaya/CreateMaterialXNodeCmd.h]:
```suggestion
/// Creates one or more MaterialX nodes from the specified MaterialX document.
```

**Pull request merged**